man/src/network/create.md

```diff
@@ -134,3 +134,44 @@ $ docker network create -d overlay \
   --opt encrypted=true \
   my-ingress-network
 ```
+
+### Run services on predefined networks
+
+You can now create services on the predefined docker networks `bridge` and `host`.
+
+```bash
+$ docker service create --name my-service \
+  --network host \
+  --replicas 2 \
+  busybox top
+```
+
+### Swarm networks with local scope drivers
+
+You can now create a swarm network with local scope network drivers. You do so
+by promoting the network scope to `swarm` during the creation of the network. 
+You will then be able to use this network when creating services. 
+
+```bash
+$ docker network create -d bridge \
+  --scope swarm \
+  --attachable \
+  swarm-network
+```
+
+For network drivers which provide connectivity across hosts (ex. macvlan), if
+node specific configurations are needed in orther to plumb the network on each
+host, you will supply that configuration via a configuration only network.
+When you create the swarm scoped network, you will then specify the name of the 
+network which contains the configuration.
+
+
+```bash
+node1$ docker network create --config-only --subnet 192.168.100.0/24 --gateway 192.168.100.115 mv-config
```

Comment:
@thaJeztah the real problem is that few drivers (such as macvlan) requires node level configuration for each of the multi-host networks that services want to make use of. Today, we dont have a clean way to orchestrate such node-network combination configurations from the manager. Hence we came up with this idea of keeping the node level network config in the Node and let the swarm handle the actual network management life-cycle (including creating and maintaining the ID) and service life-cycle (including dynamically creating the network on a node when required).
As @aluzzardi suggested, when we work towards the swarm-default feature and make it a reality, this can be addressed along with such a node/network configuration and entirely managed by the manager. In the meantime, we need such configuration as proposed in this PR. Lets make sure we have a cleaner command structure and flags in order to get this functionality.